### DISCUSSION TITLE
How to check if rules got hit ?

### DISCUSSION THREAD

**Discussion by buzzlightyear2k** [jertel/elastalert2#1511] [Q&A] [answered]
Im running a elastalert2 container, how can i check if my rule got hit or an alert is sent ? I checked my container logs it only shows:

```
Index elastalert_status already exists. Skipping index creation.
Reading Elastic 8 index mappings:
Reading index mapping 'es_mappings/8/silence.json'
Reading index mapping 'es_mappings/8/elastalert_status.json'
Reading index mapping 'es_mappings/8/elastalert.json'
Reading index mapping 'es_mappings/8/past_elastalert.json'
Reading index mapping 'es_mappings/8/elastalert_error.json'
Index elastalert_status already exists. Skipping index creation.
```

I tried to set 

```
alert:
- debug
realert:
  minutes: 1
```
but the log still the same

**Comment by buzzlightyear2k** [comment]
@jertel bro please help me

**Comment by CaGix22** [comment, marked as answer]
Hi Buzz! When you execute the command to get the elastalert2 container running, add "`--verbose`" at the end of that command. It will start logging all your rule's and ElastAlert's behavior.

See docs for more details: [ElastAlert Docs - --verbose](https://elastalert2.readthedocs.io/en/latest/logging.html)

Hope this helps!
